tutorial/create-flow.md

```diff
@@ -6,11 +6,11 @@ In the end your Flow will look similar to the following image:
 
 ![A screen shot of the completed Flow](./images/flow-team1.png)
 
-Open [Microsoft Flow](https://flow.microsoft.com) in your browser and sign in with your Office 365 tenant administrator account. Choose **My Flows** in the left-hand navigation. Choose **New**, then **Instant--from blank**. Enter `Create Team` for **Flow name**, then select **Manually trigger a flow** under **Choose how to trigger this flow**. Choose **Create**.
+Open [Microsoft Power Automate](https://flow.microsoft.com) in your browser and sign in with your Office 365 tenant administrator account. Choose **My Flows** in the left-hand navigation. Choose **New**, then **Instant--from blank**. Enter `Create Team` for **Flow name**, then select **Manually trigger a flow** under **Choose how to trigger this flow**. Choose **Create**.
 
 Select the **Manually trigger a flow** item, then choose **Add an input**, select **Text** and enter `Name` as the title.
 
-![A screen shot of the Manually trigger a flow trigger](./images/flow-team6.png)
+![A screen shot of the Manually trigger a flow trigger](./images/flow-team6.PNG)
 
 Choose **New step** and type `Batch` in the search box. Add the **MS Graph Batch Connector** action. Choose the ellipsis and rename this action to `Batch POST-groups`.
```

Comment:
At this point, I could not rename the action. The box for the connector had the text "Sign in to create a connection to MS Graph Batch Connector.", and a Sign in button. I'm not sure why it didn't pick up the connection created in the previous step?

Reply:
I am able to rename it. Interesting, would it be related with giving a permission from the connections?

Reply:
Not sure. Maybe something odd on my end.